ScoutSuite/providers/aws/facade/utils.py

```diff
@@ -0,0 +1,26 @@
+from typing import Callable
+import boto3
+
+# TODO: Add docstrings
+class AWSFacadeUtils:
+    _clients = {}
+    
+    @staticmethod
+    def get_all_pages(service, region, paginator_name: str, response_key: str, **paginator_args):
```

Comment:
I find a bit weird to specify type hints for only half of the arguments, would it be possible to add them for `service` and `region` as well?

Reply:
(I made a code suggestion just below.)

Reply:
It's indeed weird, I guess I just forgot. 🙅‍♂️